addon/services/router-scroll.js

```diff
@@ -54,19 +70,23 @@ export default Service.extend({
     set(this, 'key', stateUuid); // eslint-disable-line ember/no-side-effects
     const key = getWithDefault(this, 'key', '-1');
 
-    return getWithDefault(scrollMap, key, { x: 0, y: 0 });
+    return getWithDefault(scrollMap, key, scrollMap.default);
   }).volatile(),
 
   _loadConfig() {
     const config = getOwner(this).resolveRegistration('config:environment');
 
-    if (config && config.routerScroll && config.routerScroll.scrollElement) {
+    if (config && config.routerScroll) {
       const scrollElement = config.routerScroll.scrollElement;
-
       if ('string' === typeOf(scrollElement)) {
         set(this, 'scrollElement', scrollElement);
       }
 
+      const targetElement = config.routerScroll.targetElement;
+      if ('string' === typeOf(targetElement)) {
+        set(this, 'targetElement', targetElement);
+      }
```

Comment:
Potentially need to thinks about users that set a `scrollElement` && a `targetElement`.  I guess you could have both but that might be better solved in a follow up PR.  Maybe just an assertion/debug statement here if they do?